i18n/zh.json

```diff
@@ -7,8 +7,8 @@
     "en": "zh",
     "Paste does not exist, has expired or has been deleted.":
         "粘贴不存在，已过期或者已被删除。",
-    "%s requires php 5.3.0 or above to work. Sorry.":
-        "%s需要工作于PHP 5.3.0及以上版本，抱歉。",
+    "%s requires php 5.4.0 or above to work. Sorry.":
+        "%s需要工作于PHP 5.4.0及以上版本，抱歉。",
```

Comment:
Please replace the version number with `%s`. (see [this comment](https://github.com/PrivateBin/PrivateBin/issues/201#issuecomment-288960981))